src/registrar/views/portfolios.py

```diff
@@ -63,14 +47,11 @@ class PortfolioOrganizationView(PortfolioBasePermissionView, FormMixin):
     def get_context_data(self, **kwargs):
         """Add additional context data to the template."""
         context = super().get_context_data(**kwargs)
-        # no need to add portfolio to request context here
-        context["has_profile_feature_flag"] = flag_is_active(self.request, "profile_feature")
-        context["has_organization_feature_flag"] = flag_is_active(self.request, "organization_feature")
         return context
 
     def get_object(self, queryset=None):
         """Get the portfolio object based on the URL parameter."""
-        return get_object_or_404(Portfolio, id=self.kwargs.get("portfolio_id"))
+        return get_object_or_404(Portfolio, id=self.request.user.portfolio.id)
```

Comment:
**minor suggestion/non blocking** a function on user for get_portfolio that returns the id may make this more readable (plus I imagine getting the portfolio ID is something we would need to do often. Not a must though. Your code is so clean, I have to comment on something

Reply:
> **minor suggestion/non blocking** a function on user for get_portfolio that returns the id may make this more readable (plus I imagine getting the portfolio ID is something we would need to do often. Not a must though. Your code is so clean, I have to comment on something

So far, this is the only place where we are retrieving the portfolio id.  There are three other places in code where we are getting portfolio from user.  I am not sure yet if adding another method for getting the portfolio from the user is simpler than just user.portfolio.  Is the idea that portfolio might be None?

Reply:
Ok, decided to fix this so that it doesn't depend on the id of the portfolio.  It just returns the portfolio, which is simpler.  And still raises 404 if portfolio is none